### ISSUE TITLE
Re-add CMake compatible sort_qmake_file_entries.py script

### ISSUE THREAD

**Issue by dbrgn** [LibrePCB/LibrePCB#908]
As mentioned in https://github.com/LibrePCB/LibrePCB/pull/798#pullrequestreview-684494480, once the CMake PR (#798) branch is merged, we should re-add the `sort_qmake_file_entries.py` script.

Comment from @ubruhin:

> Btw, I just realized that with removing `sort_qmake_file_entries.py` the *.qrc files are now no longer sorted as well. So it would be really nice to re-add it - I gave it a try to even sort filenames in `CMakeLists.txt` as well, feel free to copy it from [here](https://gist.github.com/ubruhin/109312152d2298b8adeb7fec7f1ea3d5) :wink: But I could also add it later with a separate PR if you like. My script is for sure not very elegant, but so far it seems to work fine :slightly_smiling_face: 

Blocked by #798.
